elk/utils/data_utils.py

```diff
@@ -39,14 +46,17 @@ def get_columns_all_equal(dataset: DatasetDict) -> list[str]:
     return pivot
 
 
-def held_out_split(dataset: DatasetDict) -> Dataset:
-    """Return the validation set if it exits, otherwise the test set."""
-    if "validation" in dataset:
-        return dataset["validation"]
-    elif "test" in dataset:
-        return dataset["test"]
-    else:
-        raise ValueError("No validation or test split found")
+def select_train_val_splits(raw_splits: Iterable[str]) -> tuple[str, str]:
+    """Return splits to use for train and validation, given an Iterable of splits."""
+    priorities = {
+        Split.TRAIN: 0,
+        Split.VALIDATION: 1,
+        Split.TEST: 2,
+    }
+    splits = sorted(raw_splits, key=lambda k: priorities.get(k, 100))  # type: ignore
+    assert len(splits) >= 2, "Must have train and val/test splits"
```

Comment:
This error message is no longer true: should say "must have at least two of train, val, and test splits".

Reply:
yeah